bootstrap.sh

```diff
@@ -29,6 +29,14 @@ install_manifest() {  # $1 = name of the manifest file
     curl --location --remote-name https://raw.githubusercontent.com/$MANIFESTS_SOURCE_REPOSITORY/$MANIFESTS_SOURCE_FOLDER/$1.pp --output $BOOTSTRAP_MANIFESTS_DESTINATION_FOLDER
 }
 
+function run_puppet {
+    set +e
+    puppet apply --detailed-exitcodes --verbose $@
```

Comment:
I tested the $@ locally but it was without spaces. With spaces, it just failed... Thanks !